titan_doc_8.0 (#16777)
pingcap/docs: dynamic-config.md

| `{db-name}.{cf-name}.soft-pending-compaction-bytes-limit` | The soft limit on the pending compaction bytes |
| `{db-name}.{cf-name}.hard-pending-compaction-bytes-limit` | The hard limit on the pending compaction bytes |
| `{db-name}.{cf-name}.titan.blob-run-mode` | The mode of processing blob files |

> | `{db-name}.{cf-name}.titan.min-blob-size` | The threshold at which data is stored in Titan. When the value of the data reaches this threshold, it will be stored in a Titan blob file. |

Author: ```suggestion
| `{db-name}.{cf-name}.titan.min-blob-size` | The threshold at which data is stored in Titan. Data is stored in a Titan blob file when its value reaches this threshold. |
```
